src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -137,33 +144,55 @@ class CombatLogParser extends MainCombatLogParser {
             Total damage taken ${formatThousands(this.totalDamageTaken)}`}
       />,
       <StatisticBox
-        icon={(
+        icon={<Icon icon="spell_mage_altertime" alt="Dead GCD time" />}
+        value={`${formatPercentage(deadTimePercentage)} %`}
+        label='Dead GCD time'
+        tooltip={'Dead GCD time is available casting time not used. This can be caused by latency, cast interrupting, not casting anything (e.g. due to movement/stunned), etc.'}
```

Comment:
You shouldn't put strings in curly brackets when they have no code, so this could be `tooltip="Dead GCD time is available casting time not used. This can be caused by latency, cast interrupting, not casting anything (e.g. due to movement/stunned), etc."` like regular HTML.